app/assets/stylesheets/application.scss

```diff
@@ -113,45 +115,60 @@ h5 {
 }
 
 .div-bar {
-  background-color: #37becc;
-  color: #ffffff;
+  background-color: $primary-color;
+  color: $white;
```

Comment:
Properties should be ordered background-color, border-top-left-radius, border-top-right-radius, box-shadow, color, margin, margin-top, padding